Update BugTracking derby.txt

AdrijaG opened the pull request:
Please make changes into the userid column when implementing new queries from logintable insertion onwards (since my auto id generation will be different from yours)

AdrijaG committed:
Update BugTracking derby.txt

Please make changes into the userid column when implementing new queries from logintable insertion onwards (since my auto id generation will be different from yours)

Akanksha810 reviewed (approved):
Looks good to me

AdrijaG committed:
Update BugTracking derby.txt

Akanksha810 reviewed (approved):
unique email, okay. Noted.

AdrijaG committed:
Update BugTracking derby.txt

GautamiRahane commented:
Looks good.

AdrijaG committed:
import users+login (fully complete)

project manager added (incomplete: assign to dev and create new project)
tester module added (needs modification: report new bug->adding date to database and solve nullptrExc)
